basic/src/tags/queries.ts

```diff
@@ -0,0 +1,14 @@
+import { Tag } from "wasp/entities";
+import { HttpError } from "wasp/server";
+import { type GetTags } from "wasp/server/operations";
+
+export const getTags: GetTags<void, Tag[]> = (_, context) => {
```

Comment:
```suggestion
export const getTags: GetTags<void, Tag[]> = (_args, context) => {
```

Personal preference: this way it's easier to know which argument exist and when you want to start using it, you just drop the `_`.